5.6/test/run

```diff
@@ -10,7 +10,7 @@ set -o errexit
 set -o nounset
 shopt -s nullglob
 
-IMAGE_NAME=${IMAGE_NAME-centos/mysql-56-centos7-candidate}
+IMAGE_NAME=${IMAGE_NAME-centos/mysql-57-centos7-candidate}
```

Comment:
This is wrong. I see you tried to have same test file for 5.6 and 5.7. +1

To do this properly you have available `$VERSION` variable. So this line could be

`IMAGE_NAME=${IMAGE_NAME-centos/mysql-${VERSION//.}-centos7-candidate}`